onboarding/source/controller.c

```diff
@@ -78,10 +92,11 @@ static void controllerTask(void * pvParameters) {
         
         /* USER CODE BEGIN */
         // Start light timer
+        xReturned = xReturned | xTimerStart(lightTimerHandle, 0);
```

Comment:
If the led timer starts correctly, then this line would evaluate to `1 | xTimerStart(lightTimerHandle`. So if the light timer fails to start, it would be `1 | 0` which is `1`. I'm guessing that's not the intended behaviour? I think an `&&` would make more sense here.

You could also print a message if either timer fails to start.

Reply:
Also, if you have `x = x | y`, it's cleaner to write `x |= y`